### DISCUSSION TITLE
Help to get access to Discord in Russia

### DISCUSSION THREAD

**Discussion by 1am2strong4you** [dovecoteescapee/ByeDPIAndroid#169] [Q&A]
Since today (08.10.2024) Discord is officially blocked in Russia. Any ideas how to bypass it via ByeDPIAndroid?

**Comment by TanatosXTX** [comment]
Только что поставил и все работает сразу.

**Comment by v1s7** [comment]
Сам дискорд работает и с дефолтных настроек, но чтобы работали голосовые чаты нужно:
Desync UDP: ✅
UDP fake count: 1 (можно и больше)

**Reply by 1am2strong4you** [reply]
помогло, благодарю

**Reply by ladccsFS** [reply]
Доброе, есть ли это в виде команды?

**Reply by v1s7** [reply]
> Доброе, есть ли это в виде команды?

`ciadpi -d1 -a1`
Первый параметр ставит Disorder, чего достаточно для доступа к дс
Второй параметр выставляет UDP fake count на 1, чтобы оживить голосовые

**Reply by MsNicto** [reply]
а как все настроить нубу? есть подсказка где-нибудь?

**Reply by v1s7** [reply]
> а как все настроить нубу? есть подсказка где-нибудь?

Шестерня сверху справа > UI Editor

**Reply by ChrisOverHere** [reply]
а как поставить этот udp?
